tests/NLog.UnitTests/Targets/LogReceiverWebServiceTargetTests.cs

```diff
@@ -190,10 +190,18 @@ public void NoEmptyEventLists()
             {
                 Name = "NoEmptyEventLists_wrapper"
             };
-            asyncTarget.Initialize(configuration);
-            asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
-            Thread.Sleep(1000);
-            Assert.Equal(1, target.SendCount);
+            try
+            {
+                asyncTarget.Initialize(configuration);
+                asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
+                Thread.Sleep(1000);
+                Assert.Equal(1, target.SendCount);
+            }
+            finally
+            {
+                asyncTarget.Close();
```

Comment:
edit: I guess I see the issue. The asyncTarget isn't in logmanager.configuration hence it is not closed with `LogManager.Configuration.Close();`